core/ecschema-metadata/src/test/Metadata/Constant.test.ts

```diff
@@ -332,6 +332,99 @@ describe("Constant", () => {
       assert.strictEqual(constantSerialization.definition, "PI");
       assert.strictEqual(constantSerialization.phenomenon, "TestSchema.TestPhenomenon");
     });
+
+    it("async - when only numerator property is defined", async () => {
+      const schemaJson = createSchemaJson({
+        label: "Test Constant",
+        description: "testing a constant",
+        phenomenon: "TestSchema.TestPhenomenon",
+        definition: "PI",
+        numerator: 5.5,
+      });
+
+      const ecSchema = await Schema.fromJson(schemaJson, new SchemaContext());
+      assert.isDefined(ecSchema);
+      const testItem = await ecSchema.getItem<Constant>("TestConstant");
+      assert.isDefined(testItem);
+      assert.isTrue(testItem?.schemaItemType === SchemaItemType.Constant);
+      const testConst: Constant = testItem as Constant;
+      assert.isDefined(testConst);
+      const constantSerialization = testConst.toJSON(true, true);
+
+      expect(constantSerialization.$schema).eql("https://dev.bentley.com/json_schemas/ec/32/schemaitem");
+      expect(constantSerialization.name).eql("TestConstant");
+      expect(constantSerialization.schemaVersion).eql("01.02.03");
+      expect(constantSerialization.schema).eql("TestSchema");
+
+      expect(constantSerialization.label).eql("Test Constant");
+      expect(constantSerialization.description).eql("testing a constant");
+
+      assert.strictEqual(constantSerialization.definition, "PI");
+      assert.strictEqual(constantSerialization.phenomenon, "TestSchema.TestPhenomenon");
```

Comment:
do we need to test these every time?  We are focused on just the numerator and denominator in this case

Reply:
No, I think we can just go with the 3 properties check.